### DISCUSSION TITLE
Airtable Import Maximum 25 Columns?

### DISCUSSION THREAD

**Discussion by evanmcgillivray** [AntlerVC/firetable#1292] [Support / Q&A] [answered]
I tried to import my Airtable base there is 54 colums in the table but I only see 25 when importing?
Airtable Count
<img width="396" alt="Screenshot 2023-06-11 at 9 07 17 PM" src="https://github.com/rowyio/rowy/assets/21324526/5a20d459-1b8a-4d62-a0bd-04708855c873">
Rowy Count
<img width="225" alt="Screenshot 2023-06-11 at 9 08 55 PM" src="https://github.com/rowyio/rowy/assets/21324526/760f3660-7387-4d3d-9976-0d070141527e">

**Comment by BeeBombshell** [comment, marked as answer]
Hey @evanmcgillivray, saw your post on the Discord Server as well... :)

Based on our investigation so far, it appears that the problem stems from how the columns are defined during the import process, using the first row or object as a reference. However, I want to assure you that our team is actively working on a fix, which will be released shortly to address this issue.

In the meantime, I'd like to offer a couple of suggestions to make this work. First, you can export your Airtable data as a CSV file (screenshot below). This format may provide a workaround until the fix is implemented. Alternatively, you can try importing the data again, ensuring that you fill-up the rows for the first column that needs to be imported.
![image](https://github.com/rowyio/rowy/assets/53828745/cd4162a0-8c22-4a77-a64e-9d3130af8962)

**Comment by evanmcgillivray** [comment]
Subject: Re: Response to Import Issue Subject: Re: Further Clarification Needed on Import IssueHi there,I really appreciate your prompt response and proactive approach in resolving this issue. I need a bit more information regarding the "first row trick." Specifically, I'm curious about which record is considered the first one used to identify the columns? Is there a specific sort order that's applied? Interestingly, I've noticed a similar issue on make.Com.For the CSV import, could you provide further guidance on managing linked records? What field type should these be categorized as? Moreover, would I need to modify all tables with a linked record to designate the record as the primary field? I want to make sure I have a solid understanding of these processes.Thanks again for your help with this issue!Best, Evan ***@***.*** from my iPhoneOn Jun 14, 2023, at 12:46 AM, Bhavya Verma ***@***.***> wrote:﻿
Hey @evanmcgillivray, saw your post on the Discord Server as well... :)
Based on our investigation so far, it appears that the problem stems from how the columns are defined during the import process, using the first row or object as a reference. However, I want to assure you that our team is actively working on a fix, which will be released shortly to address this issue.
In the meantime, I'd like to offer a couple of suggestions to make this work. First, you can export your Airtable data as a CSV file (screenshot below). This format may provide a workaround until the fix is implemented. Alternatively, you can try importing the data again, ensuring that you fill-up the rows for the first column that needs to be imported.
